File: src/org/fortune/craft/fragments/ChangelogActivity.java
/*
 *  Copyright (C) 2024 FortuneOS
 *  SPDX-License-Identifier: Apache-2.0
 */

package org.fortune.craft.fragments;

import android.os.Bundle;
import com.android.settingslib.collapsingtoolbar.CollapsingToolbarBaseActivity;
import com.android.settingslib.widget.R;

public class ChangelogActivity extends CollapsingToolbarBaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getFragmentManager().beginTransaction()
                .replace(R.id.content_frame, new ChangelogFragment())
                .commit();
    }
}

File: src/org/fortune/craft/category/ChangelogActivity.kt
package org.fortune.craft.category

import android.os.Bundle
import com.android.settingslib.collapsingtoolbar.CollapsingToolbarBaseActivity

class ChangelogActivity : CollapsingToolbarBaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        fragmentManager.beginTransaction().replace(
            com.android.settingslib.collapsingtoolbar.R.id.content_frame,
            ChangelogFragment()
        ).commit()
    }
}
